docs/main_text.md

```diff
@@ -60,31 +60,148 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
+The platform with the highest overlap (provided it is >75%) is inferred to be the true platform.
+Some analyses around this platform detection procedure can be found in [this repository](https://github.com/jaclyn-taroni/beadarray-platform-detection).
+
+### RNA-seq pipelines
+
+We use [Salmon](https://combine-lab.github.io/salmon/) and [tximport](https://bioconductor.org/packages/release/bioc/html/tximport.html) to process all RNA-seq data in refine.bio.
+We obtain fastq files run on our [supported short-read platforms](https://github.com/AlexsLemonade/refinebio/blob/dev/config/supported_rnaseq_platforms.txt) from Sequence Read Archive. 
+We use the library strategy and library source metadata fields to identify RNA-seq experiments.
+It's possible that experiments that are inappropriate for use with Salmon will still appear in refine.bio (e.g., long-read platforms that are labeled incorrectly in the source repository).
+If you find an experiment that you believe is inappropriate for use with Salmon, please [file an issue on GitHub](https://github.com/AlexsLemonade/refinebio/issues) so that we can resolve it.
+If you would prefer to report issues via e-mail, you can also email [ccdl@alexslemonade.org](mailto:ccdl@alexslemonade.org).
+
+#### Salmon 
+
+Salmon is an alignment-free method for estimating transcript abundances from RNA-seq data ([Patro, et al. _Nature Methods_. 2017.](http://dx.doi.org/10.1038/nmeth.4197)). 
+We use it in [quasi-mapping mode](http://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-quasi-index-and-fmd-index-based-modes), which is significantly faster than alignment-based approaches and requires us to build a Salmon transcriptome index. 
+
+##### Transcriptome index
+
+We build a custom reference transcriptome (using [RSEM](https://github.com/deweylab/RSEM) `rsem-prepare-reference`) by filtering the Ensembl genomic DNA assembly to remove _pseudogenes_, which we expect could negatively impact the quantification of protein-coding genes. 
+This means we're obtaining abundance estimates for coding as well as non-coding transcripts. 
+
+Building a transcriptome index with `salmon index` requires us to specify a value for the parameter `-k` that determines the size of the k-mers used for the index.
+The length of a read determines what k-mer size is appropriate. 
+Consistent with the recommendations of the authors of Salmon, we use an index build with _k_ = 31 when quantifying samples with reads with length > 75bp.
+We use _k_ = 23 for all other read lengths.
+
+TODO: Note about how to obtain Salmon indices. Do we also make gene to transcript mapping passed to `rsem-prepare-reference` available?
+
+##### Quantification with Salmon
+
+When quantifying transcripts with `salmon quant`, we take advantage of options that allow Salmon to learn and attempt to correct for certain biases in sequencing data. 
```

Comment:
instead of `learn` maybe `predict`? 🤷‍♂️  Not a strong feeling on this one

Reply:
I left it as is because it's consistent with the Salmon documentation. `model` might be a better word here?